server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -3010,7 +3010,7 @@ public ReplicationStats getReplicationStats() {
             long maxBytesBehind = stats.stream().mapToLong(SegmentReplicationShardStats::getBytesBehindCount).max().orElse(0L);
             long totalBytesBehind = stats.stream().mapToLong(SegmentReplicationShardStats::getBytesBehindCount).sum();
             long maxReplicationLag = stats.stream()
-                .mapToLong(SegmentReplicationShardStats::getCurrentReplicationTimeMillis)
+                .mapToLong(SegmentReplicationShardStats::getCurrentReplicationLagMillis)
```

Comment:
nit: Do we need tests around this ?
```suggestion
                .mapToLong(SegmentReplicationShardStats::getCurrentReplicationLagMillis)
```

Reply:
We're just changing which replication lag we return. There's no way to check the value, and the flow is handled by existing tests so we should be covered.